frontend/packages/console-shared/src/components/dashboard/dashboard-card/card.scss

```diff
@@ -1,6 +1,8 @@
 @import '~@patternfly/patternfly/sass-utilities/colors';
 
 $co-dashboard-card--line-height: 19px;
+$co-dashboard-icon-font-size: 1.2rem;
```

Comment:
As we discussed, I think it's better for this to be renamed to move to `_icon-and-text.scss` and be imported here rather than the other way around.

Reply:
updated pr